pkg/virt-config/feature-gates.go

```diff
@@ -58,6 +64,20 @@ func (c *ClusterConfig) isFeatureGateEnabled(featureGate string) bool {
 	return false
 }
 
+func (c *ClusterConfig) IsFeatureGateDeprecated(featureGate string) bool {
+	deprecatedFeatureGates := [...]string{
```

Comment:
what about defining this with a compilation-unit scope (it can still be private) instead of hiding it inside a function?

Reply:
I didn't mean making the list public, I'd still keep it private, I'd just give it a file-scope instead of a function-scope for better visibility inside the file.

Reply:
yeah, it would be nice to have this list around the feature gate `const` values at the top of the file.

Reply:
> yeah, it would be nice to have this list around the feature gate `const` values at the top of the file.

Can you explain what the advantage is?
Do we want people to refer to the list explicitly instead of calling the function? If not I just don't see how it helps

Reply:
> Can you explain what the advantage is?

you're right there is not an immediate technical advantage (that i'm aware of).

It's more of a coding convention that i would expect to see that the const data is separate from the helper function.